src/ImageSharp/Formats/Gif/LzwDecoder.cs

```diff
@@ -73,7 +78,7 @@ public void DecodePixels(int minCodeSize, Buffer2D<byte> pixels)
         // It is possible to specify a larger LZW minimum code size than the palette length in bits
         // which may leave a gap in the codes where no colors are assigned.
         // http://www.matthewflickinger.com/lab/whatsinagif/lzw_image_data.asp#lzw_compression
-        if (minCodeSize < 2 || clearCode > MaxStackSize)
+        if (minCodeSize < 2 || minCodeSize > MaximumLzwBits || clearCode > MaxStackSize)
```

Comment:
To save a comparison:
```suggestion
        if ((uint)minCodeSize - 2 > MaximumLzwBits - 2 || clearCode > MaxStackSize)
```
(due the `Unsafe.Add` below I guess this method is hot enough...)

Reply:
thanks for the suggestions @gfoidl, but I do not see any difference when I execute gif decoder benchmark.

I do not want to sacrifice the readability here, if it does not really improve the performance.